### PR TITLE
feat: [FFM-6971] : Adding jira issues to feature flag detail

### PR DESCRIPTION
**Pull request by chrisblakely01**
### Summary
Renders a list of Jira Issues for a given feature flag

#### Screenshots

Screenshot 2023-02-23 at 15 54 37

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by chrisblakely01** [comment]
retrigger titlecheck

**Comment by chrisblakely01** [comment]
retrigger titlecheck

File: src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <Heading level={5} font={{ variation: FontVariation.H5 }}>
+          {getString('cf.featureFlagDetail.jiraIssuesTitle')}
+        </Heading>
+        <Text font={{ variation: FontVariation.SMALL_SEMI }} color={Color.GREY_500}>
+          {getString('cf.featureFlagDetail.jiraIssuesDescription')}
+        </Text>
+      </div>
+
+      {jiraIssues.map(issue => (
+        <a key="issue.key" href={issue.issueURL} target="_blank" rel="noreferrer">
```
Comment (knagurski) [inline src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:26]:
```suggestion
        <a key={issue.key} href={issue.issueURL} target="_blank" rel="noreferrer">
```

Comment (knagurski) [inline src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:26]: I _think_ it's best practice to include `noopener` too

File: src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
+    >
+      <JiraIssueList jiraIssues={jiraIssues} {...props} />
+    </TestWrapper>
+  )
+}
+
+describe('JiraIssueList', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
```
Comment (knagurski) [inline src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx:38]: Reset is fine in NGUI, but when we plop this in FF-UI it'll kill the `TestWrapper` (the test wrapper mocks things in FF-UI and is all hard-coded in NGUI). In this specific case, you're not mocking anything, so you can drop it altogether, but in general favour `clearAllMocks` over `resetAllMocks`
```suggestion
    jest.clearAllMocks()
```

File: src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <div>
+        <Heading level={5} font={{ variation: FontVariation.H5 }}>
+          {getString('cf.featureFlagDetail.jiraIssuesTitle')}
+        </Heading>
+        <Text font={{ variation: FontVariation.SMALL_SEMI }} color={Color.GREY_500}>
+          {getString('cf.featureFlagDetail.jiraIssuesDescription')}
+        </Text>
+      </div>
+
+      {jiraIssues.map(issue => (
```
Comment (knagurski) [inline src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:25]: We probably want some kind of empty state when there are no issues... maybe `"No associated Jira issues"` or something to that effect.

Comment (chrisblakely01) [reply src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:25]: thought the same, but the prereqs and servicelist dont have an empty state (just the button) so would be inconsistent. Probs something to address as a whole later
 
Screenshot 2023-02-27 at 11 52 10

File: src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx (hunk 1 above)
Comment (chrisblakely01) [reply src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:26]: fixed

Comment (chrisblakely01) [reply src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:26]: added

File: src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx (hunk 2 above)
Comment (chrisblakely01) [reply src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx:38]: removed

File: src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx (hunk 3 above)
Comment (knagurski) [reply src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:25]: Yeah, these do need standardised. Not great that the 3 of them behave differently. One for another ticket though. We can ping Ksenia to get a solution 👍🏻

Comment (chrisblakely01) [reply src/modules/75-cf/components/FlagActivation/JiraIssueList.tsx:25]: sounds good 👍

**Review by knagurski — APPROVED** [review]
Nice 👌

**Review by laura-mannus — APPROVED** [review]
Really nice

**Pull request merged**